entrypoint.sh

```diff
@@ -98,11 +98,11 @@ fi
 
 echo "----------------------------------"
 echo "${OUTPUT}"
-echo "::set-output name=output::${OUTPUT}"
-echo "::set-output name=report::gitleaks-report.${INPUT_REPORT_FORMAT}"
-echo "::set-output name=result::${GITLEAKS_RESULT}"
-echo "::set-output name=command::${command}"
-echo "::set-output name=exitcode::${exitcode}"
+echo "output=${OUTPUT}" >>$GITHUB_OUTPUT
+echo "report=gitleaks-report.${INPUT_REPORT_FORMAT}" >>$GITHUB_OUTPUT
+echo "result=${GITLEAKS_RESULT}" >>$GITHUB_OUTPUT
+echo "command=${command}" >>$GITHUB_OUTPUT
+echo "exitcode=${exitcode}" >>$GITHUB_OUTPUT
 
 echo "Gitleaks Summary: ${GITLEAKS_RESULT}" >>$GITHUB_STEP_SUMMARY
```

Comment:
same comment here, I think the above variable would now be saved in `${{ env.GITLEAKS_RESULT }}` after the changes

Reply:
its built in GH feature: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#adding-a-job-summary